lib/ansible/modules/network/aci/aci_tenant.py

```diff
@@ -20,161 +20,217 @@
                     'status': ['preview'],
                     'supported_by': 'community'}
 
-
-DOCUMENTATION = '''
+DOCUMENTATION = r'''
 ---
-
 module: aci_tenant
-short_description: Direct access to the APIC API
+short_description: Manage tenants on Cisco ACI fabrics
 description:
-    - Offers direct access to the APIC API
-author: Cisco
-extends_documentation_fragment: aci
+- Manage tenants on a Cisco ACI fabric.
+author:
+- Swetha Chunduri (@schunduri)
+- Dag Wieers (@dagwieers)
 requirements:
-    - ACI Fabric 1.0(3f)+
-notes:
+- ACI Fabric 1.0(3f)+
 options:
-    tenant_name:
-        description:
-            - Tenant Name
-        required: true
-        default: null
-        choices: []
-        aliases: []
-    descr:
-        description:
-            - Description for the AEP
-        required: false
-        default: null
-        choices: []
-        aliases: []
+  method:
+    description:
+    - The HTTP method of the request.
+    - Using C(delete) is typically used for deleting objects.
+    - Using C(get) is typically used for querying objects.
+    - Using C(post) is typically used for modifying objects.
+    required: true
+    default: get
+    choices: [ delete, get, post ]
+    aliases: [ action ]
+  tenant_name:
+    description:
+    - The name of the tenant.
+    required: yes
+  descr:
+    description:
+    - Description for the AEP.
+extends_documentation_fragment: aci
 '''
 
 EXAMPLES = '''
-
-    aci_tenant:
-       action: "{{ action }}"
-       tenant_name: "{{ tenant_name }}"
-       descr: "{{ descr }}"
-       host: "{{ host }}"
-       username: "{{ user }}"
-       password: "{{ pass }}"
-       protocol: "{{ protocol }}"
-
+- name: Add a new tenant
```

Comment:
More real-life examples needed, things that show good naming convention and/or best-practices.
Also should demonstrate how to query information and/or delete a tenant.